### PR TITLE
Improvements in the rotation between continuous particles

### PR DESCRIPTION
**Pull request by joaquinirazabal**
1. After some research, the calculation of the rotational moments for continuum particles is improved. This was tested by developing some calculations of a cantilever beam.

2. The relative displacement due to particles rotation is calculated with quaternions.

3. The global damping function includes the damping of the torques and all forces, not only contact forces

### CODE REVIEW

**Comment by joaquinirazabal** [comment]
Yes, @maceligueta I realized it just when I push the PR. Fixed ;)

File: applications/DEM_application/custom_elements/spheric_continuum_particle.cpp
Comment (maceligueta) [inline applications/DEM_application/custom_elements/spheric_continuum_particle.cpp]: The whole word Quaternion would be fine as well

**Comment by maceligueta** [comment]
Looks fine. @salvalatorre  coded the previous version of that, so I'll let him approve.

**Review by salvalatorre — DISMISSED** [review]
Looks OK to me!

**Review by salvalatorre — APPROVED** [review]
Looks like you last commit dismissed my first approval...

**Comment by joaquinirazabal** [comment]
You are right! Thanks @salvalatorre

**Comment by joaquinirazabal** [comment]
Merged

**Pull request merged**
